spec/index.bs

```diff
@@ -3149,9 +3132,28 @@ enum GPUBufferMapState {
     : <dfn>usage</dfn>
     ::
         The allowed usages for this {{GPUBuffer}}.
+</dl>
 
-    : <dfn>\[[internals]]</dfn>, of type [=buffer internals=], readonly, {{GPUObjectBase/[[internals]]|override}}
+{{GPUBuffer}} has the following [=device timeline properties=]:
+
+<dl dfn-type=attribute dfn-for=GPUBuffer data-timeline=device>
+    : <dfn>\[[internalState]]</dfn>
```

Comment:
nit: I think we're sort of moving toward `[[internal state]]` as the naming style for slots.

non-blocking aside: Maybe I want to call it `[[device timeline state]]`, though that naming is very clunky. (Maybe I wish we had a better name for "device timeline" - it kinda makes sense in the spec because it's associated with a particular GPUDevice, but really what we're trying to communicate is "gpu process timeline" / "service-side timeline". Could be "internal timeline", then `[[internal state]]` makes more sense?)

Reply:
Renamed to `[[internal state]]`

![which-was-the-style-at-the-time-the-simpsons](https://github.com/gpuweb/gpuweb/assets/805273/c9bb724e-9218-4e1e-b400-a5729d2822a7)

bikeshedding about device timeline/internal timeline/gpu process timeline/etc left for another day.